### ISSUE TITLE
please let know the dependent library

### ISSUE THREAD

**Issue by zaykl** [tum-vision/fastfusion#19]
I would to run this code. But I can't find any detail about the dependent library. Such as the version of opencv. Pls let know. That will help me a lot.

**Comment by Pratapabhay** [comment]
Hi @zaykl  follow the instructions in the Readme File. While running cmake it will notify you what all libraries you are missing. Just Install them and run the make command, you are good to go.

**Comment by zaykl** [comment]
@Pratapabhay I don't think this is a good idea. Sometimes the latest version library may cause crash.

**Comment by moonfish930** [comment]
I build the project on VMware Ubuntu14, here is my process
# opencv
```bash
sudo apt-get install cmake  
sudo apt-get install build-essential cmake libgtk2.0-dev pkg-config Python-dev python-numpy libavcodec-dev libavformat-dev libswscale-dev 
```
then download opencv2.4 on https://github.com/opencv/opencv;  build and install opencv2.4 
# qt4
first I remove qt5 thoroughly, then install qt4 through apt-get, at last install QGLviewer2.6.3(this is a version matches qt4)
```bash
sudo apt-get purge --auto-remove qt5-default
sudo apt-get install qt4-dev-tools qt4-doc qt4-qtconfig qt4-demos qt4-designer
```
download QGLvierwer2.6.3 from http://libqglviewer.com/src/  , then install it
# others
```bash
sudo apt-get install libeigen3-dev libboost-all-dev libglew-dev 
```
# source code modify
in CMakeLists.txt
```cmake
#find_package(Eigen3 REQUIRED)  
include_directories(EIGEN3_INCLUDE_DIR)  
include_directories("/usr/include/eigen3")
```
in src/CMakeLists.txt
```cmake
message(STATUS "Did not find qglviewer2, linking qglviewer-qt4")
set(QGLVIEWER QGLViewer)
```
in onlinefusionviewer.cpp
```c++
#include <QGLViewer/manipulatedFrame.h>
```
